protocol-tests/src/main/java/fixtures/paging/implementation/PagingsImpl.java

```diff
@@ -860,7 +860,7 @@ Mono<Response<BinaryData>> getPagingModelWithItemNameWithXmsClientNameNext(
      * @return the response body along with {@link PagedResponse} on successful completion of {@link Mono}.
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
-    private Mono<PagedResponse<BinaryData>> getNoItemNamePagesSinglePageAsync(RequestOptions requestOptions) {
+    public Mono<PagedResponse<BinaryData>> getNoItemNamePagesSinglePageAsync(RequestOptions requestOptions) {
```

Comment:
Why is this generated as `public`?

Reply:
I forgot to rebuild after pulling in main, will regenerate